plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/EclipseProjectCreator.java

```diff
@@ -0,0 +1,21 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IWorkspaceRoot;
+import org.eclipse.core.resources.ResourcesPlugin;
+
+/**
+* Utility to make a new Eclipse project with the App Engine Standard Nature in the workspace.  
+*/
+class EclipseProjectCreator {
+
+  /**
+   * @return true on successful project creation; false otherwise
+   */
+  static boolean makeNewProject(AppEngineStandardProjectConfig config) {
```

Comment:
IStatus sounds good. done and done. Will flesh this out further as the EclipseProjectCreator grows.